README.md

```diff
@@ -1,24 +1,203 @@
-# Angular Training Slides
+# Angular Training Materials
 
-Slides for the Angular 2+ training created with Reveal.js and using Markdown format
+This repository contains slides and other materials
```

Comment:
No, I just prefer semantic linebreaks (leads to fewer errors when moving chunks of text around). Markdown renders all consecutive lines w/out separating blanks as one para.